src/OSPSuite.Core/Domain/PKAnalyses/DynamicPKParameter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System.Linq;
+using OSPSuite.Core.Domain.Services;
+
+namespace OSPSuite.Core.Domain.PKAnalyses
+{
+   public enum StandardPKParameter
+   {
+      Unknown,
+      Cmax,
+      Cmin,
+      Tmax,
+      Tmin,
+      CTrough,
+      Auc,
+      Aucm,
+      AucInf,
+      AucTendInf,
+      Mrt,
+      FractionAucEndToInf,
+      Thalf,
+      Vss,
+      Vd,
+      Tthreshold
+   }
+
+   public class DynamicPKParameter : PKParameter
+   {
+      // Parameter such as AUC, Cmax, Tmax that will be used to calculate the parameter
+      public StandardPKParameter StandardPKParameter { get; set; } = StandardPKParameter.Unknown;
+
+      public float? StartTime { get; set; }
+      public float? EndTime { get; set; }
+
+      /// <summary>
+      ///    1-based index of the application that should be used to estimate the start of the interval
+      /// </summary>
+      public int? StartApplication { get; set; }
```

Comment:
would rename it to StartApplication**Index**

Reply:
Maybe it should be zero based in .net and r needs to add +1?